perfdash/config.go

```diff
@@ -188,6 +190,19 @@ func getProwConfig() (Jobs, error) {
 				thisPeriodicConfig.Descriptions = jobTypeToDescriptions[jobType]
 				continue
 			}
+			if strings.HasPrefix(tag, "perfDashBuilds:") {
+				split := strings.SplitN(tag, ":", 2)
+				i, err := strconv.Atoi(strings.TrimSpace(split[1]))
+				if err != nil {
+					fmt.Fprintf(os.Stderr, "warning: unparsable builds - %v\n", split[1])
+				}
+				if i < 1 {
```

Comment:
Shouldn't this be 0 (if you're excluding negative values)?

Reply:
Or change the error msg below.

Reply:
You are right. Should be non-negative.